src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/DotNetHelper.cs

```diff
@@ -284,6 +284,15 @@ private static bool DetermineIsMonoRuntime(string dotnetRoot)
         return File.Exists(Path.Combine(sharedFramework, "mono-gc.h"));
     }
 
+    public bool IsActionPublishComplex() {
```

Comment:
To me `Should` is more meaningful than `IsAction`

```suggestion
    public bool ShouldPublishComplex() {
```